README.md

```diff
@@ -67,6 +67,10 @@ CarbonData is built using Apache Maven, to [build CarbonData](https://github.com
 * [Carbon as Spark's Datasource](https://github.com/apache/carbondata/blob/master/docs/carbon-as-spark-datasource-guide.md) 
 * [FAQs](https://github.com/apache/carbondata/blob/master/docs/faq.md) 
 
+##  Integrations
```

Comment:
Need to update in documentation.md and in quick-start-guide.md
The website carbondata.apache.org picks from these two md files

Reply:
OK, I will update the documents at soon

Reply:
@sraghunandan  I have  updated the  relevant documents : documentation.md and in quick-start-guide.md, please check it if you have free time